microsoft-edge/webview2/how-to/runtime-channel.md

```diff
@@ -0,0 +1,188 @@
+
+1.  Find `CreationProperties.BrowserExecutableFolder`.  For example:
+
+    ```csharp
+    WebView2 GetReplacementControl(bool useNewEnvironment)
+    {
+        WebView2 replacementControl = new WebView2();
+        ((System.ComponentModel.ISupportInitialize)(replacementControl)).BeginInit();
+        // Setup properties and bindings.
+        if (useNewEnvironment)
+        {
+            // Create a new CoreWebView2CreationProperties instance so the environment
+            // is made anew.
+            replacementControl.CreationProperties = new CoreWebView2CreationProperties();
+            replacementControl.CreationProperties.BrowserExecutableFolder = webView.CreationProperties.BrowserExecutableFolder;
+            replacementControl.CreationProperties.Language = webView.CreationProperties.Language;
+            replacementControl.CreationProperties.UserDataFolder = webView.CreationProperties.UserDataFolder;
+            shouldAttachEnvironmentEventHandlers = true;
+        }
+    ```
+
+### WPF
+
+WPF uses an approach similar to the Win32/C++ approach described above.
+
+See [CoreWebView2CreationProperties.BrowserExecutableFolder Property](https://docs.microsoft.com/dotnet/api/microsoft.web.webview2.wpf.corewebview2creationproperties.browserexecutablefolder?view=webview2-dotnet-1.0.864.35#Microsoft_Web_WebView2_Wpf_CoreWebView2CreationProperties_BrowserExecutableFolder).
+
+
+<!--========================================================================-->
+## Using a group policy
+
+To make your application use the Canary or Dev channel of Microsoft Edge by using a group policy, you copy ADMX and ADML files to PolicyDefinitions, as follows.
+
+1.  Get the policy files.  To do this, download policy files from [Download and deploy Microsoft Edge for business](https://www.microsoft.com/edge/business/download).  Or, find generated policy files in `out\debug_x64\gen\chrome\app\policy\windows\admx` after running `autoninja edge_webview2_policy_templates`.
+
+1.  Copy the ADMX file into a Policy Definitions template folder, such as `C:\Windows\PolicyDefinitions`.
+
+1.  Copy the ADML file into a matching locale folder within the `Policy Definitions` folder, such as a `C:\Windows\PolicyDefinitions\en-us` folder.
+
+1.  Configure policies using the **Local Group Policy Editor** or by using command-line tools (see below).<!--below, where?-->
+
+    The registry location is: `{HKLM/HKCU}\Software\Policies\Microsoft\Edge\WebView2\{override name}`
+
+For more information, see [Configure Microsoft Edge policy settings](https://docs.microsoft.com/DeployEdge/configure-microsoft-edge).
+
+<!-- link to Enterprise docs re: Group Policy? -->
+
+
+<!--========================================================================-->
+## Using a registry override
```

Comment:
Browser executable folder is one override, another is release channel preference. This applies to registry override, group policy, and env variable - you have both options